.github/workflows/push-tags.yml

```diff
@@ -43,8 +43,17 @@ jobs:
         IMAGE=gcr.io/${{ env.PROJECT_ID }}/sandbox/loadgenerator/gke:${{ steps.get_version.outputs.VERSION }}
         docker build -t $IMAGE ./src/loadgenerator
         docker push $IMAGE
+    - name: Install Hugo and npm
+      timeout-minutes: 20
+      run: sudo apt-get install hugo npm
+    - name: Install PostCSS
+      timeout-minutes: 20
+      run: sudo npm install -g autoprefixer postcss-cli postcss
```

Comment:
I added these dependencies to the GitHub Actions worker VM. Could you add them to the [dependencies script there](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/master/.github/workflows/install-dependencies.sh)?

```
# install nodejs/hugo
curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
sudo apt-get install -y nodejs hugo
sudo npm install -g autoprefixer postcss-cli postcs
```

Reply:
On it now, will get it done this afternoon